src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx

```diff
@@ -281,12 +284,13 @@ class UploadDetachedFileValidation extends React.Component {
 			requestName: 'detached_award',
 			progress: '0'
 		}
+		
 		validationBox = <ValidateDataFileContainer type={type} data={this.state.jobResults}/>;
 		if (!this.state.headerErrors && this.state.validationFinished) {
-			validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} setUploadItem={this.uploadFile.bind(this)} updateItem={this.uploadFile.bind(this)} published={this.state.published}/>;
+			validationBox = <ValidateValuesFileContainer type={type} data={this.state.jobResults} setUploadItem={this.uploadFile.bind(this)} updateItem={this.uploadFile.bind(this)} published={this.state.published} rows={this.state.fabs_meta} />;
 			if(this.state.published){
 				// This submission is already published and cannot be republished
-				validationButton = <button className='pull-right col-xs-3 us-da-disabled-button' disabled>File Already Published</button>;
+				validationButton = <button className='pull-right col-xs-3 us-da-disabled-button' disabled>File Published:<span className='plain'> {this.state.fabs_meta.valid_rows} rows published on {this.state.fabs_meta.publish_date}</span></button>;
```

Comment:
publish_date is in UTC here. I just published a submission (11:42 my time) and it shows that it was published at 3:24PM